aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -32,8 +34,21 @@ module aptos_std::simple_map {
         create()
     }
 
+    /// Create a SimpleMap from a vector of keys and values. The keys must be unique.
+    public fun new_from<Key: store, Value: store>(
+        keys: vector<Key>,
+        values: vector<Value>,
+    ): SimpleMap<Key, Value> {
+        let map = SimpleMap {
+            data: vector::empty(),
+        };
+        add_all(&mut map, keys, values);
+        map
+    }
+
     #[deprecated]
     /// Create an empty SimpleMap.
+    /// This function is deprecated, use `new` instead.
```

Comment:
Thank you!! Every time I instantiate something in the stdlib it's a real shot in the dark whether you should use `new`, `empty`, or `create`.